tests/harness/executor/BUILD

```diff
@@ -59,5 +59,5 @@ go_binary(
         shard_count = 5,
         deps = ["@bazel_tools//tools/bash/runfiles"],
     )
-    for lang in ("cc", "go", "java", "python")
+    for lang in ("cc", "java", "python")
```

Comment:
I don't really understand bazel at all, so probably I'm just missing something and sorry if that's a dumb question, but just in case, is this change intentional or some fallout from local testing of the PR?

Reply:
Yeah I was wondering about this too.

It came from the revert: https://github.com/envoyproxy/protoc-gen-validate/pull/465/files#diff-73fd757acad8e60cd3f03b49c43d67cb4d7e72b80cda6b481c6380de365045b8R62

Though some of the revert was done manually by @akonradi. Let me unwind where it came from to make sure it's a valid change.